Src/DependencyCollector/Shared.Tests/Implementation/ProfilerHttpProcessingTest.cs

```diff
@@ -938,12 +953,12 @@ private void Initialize(string instrumentationKey)
                 new List<string>(),
                 RandomAppIdEndpoint);
 
-            var correlationIdLookupHelper = new CorrelationIdLookupHelper((string ikey) =>
+            var correlationIdLookupHelper = new CorrelationIdLookupHelper(new Dictionary<string, string>
             {
-                // Pretend App Id is the same as Ikey
-                var tcs = new TaskCompletionSource<string>();
-                tcs.SetResult(ikey);
-                return tcs.Task;
+                {
+                    instrumentationKey,
+                    "cid-v1:" + instrumentationKey
```

Comment:
Ideally - do `"cid-v1:" + instrumentationKey + "-appid"` so test will validate that code doesn't use instrumentationKey mistakenly